internal/pkg/externalplugins/blunderbuss/blunderbuss.go

```diff
@@ -102,6 +103,15 @@ func HandlePullRequestEvent(gc githubClient, pe *github.PullRequestEvent,
 	repo := &pe.Repo
 	opts := cfg.BlunderbussFor(repo.Owner.Login, repo.Name)
 
+	// Wait a few seconds to allow other automation plugin to apply labels.
+	gracePeriod := 5 * time.Second
+
+	if opts.GracePeriodDuration != 0 {
```

Comment:
I think we should validate this configuration item in config.go.

It can not less than zero.